diff-iam-policy.yaml

```diff
@@ -0,0 +1,32 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: governmentpaas/awscli
+params:
+  AWS_ACCESS_KEY_ID: ((secrets.role-putter-access-key-id))
+  AWS_SECRET_ACCESS_KEY: ((secrets.role-putter-secret-access-key))
+inputs:
+- name: policies
+run:
+  path: sh
+  args:
+  - -ec
+  - |
```

Comment:
Not finished? Need to remove the script from the YAML and run the separated script instead

Reply:
Forgot to commit yesterday. Have now changed to run scripts